pagopa/io-app
# fix: [IABT-1319,IOAPPCIT-8] Local login notification scheduling removed

## Short description
This PR removes the scheduling of local login notifications during the application startup.

## List of changes proposed in this pull request
All the code related to scheduling or cancelling local notifications has been removed, notification content translations included (this was possibile thanks to the fact that local notifications were used just for this login check).
Local scheduled notification removal has been moved into the startup saga. This is needed to make sure that already-scheduled-notifications for existing users are removed upon application upgrade. 
Such removal code has been moved into the startup saga in order to be clearly noticeable for future developments (and removals - once the new application version has been extensively adopted).

## How to test
Start the application and make sure that there are no runtime error or crashes.

An in-depth test requires that you deploy a modified previous-version of the application, where a lot of notifications have been scheduled (_i.e._, edit the code to schedule 50 notifications every 10 seconds). Start this custom version and make sure that the device/emulator displays the notifications. Put the application in the background and perform an upgrade to this new version. Local notifications should disappear and future scheduled ones should not appear.

Also make sure that received remote (push) notifications are not deleted upon application start up (so make sure that the application was not running in the background before launching it).
